lib/spaceship/client.rb

```diff
@@ -170,7 +179,10 @@ def login(user = nil, password = nil)
       self.user = user
 
       begin
-        send_login_request(user, password) # different in subclasses
+        @loggedin = false
```

Comment:
when we rescue `UnauthorizedAccessError` I don't want to retry if I am not logged in. This is to avoid getting into a loop with e.g. invalid credentials.